garage/logger/tensorboard_inputs.py

```diff
@@ -0,0 +1,76 @@
+"""Contains input classes for tensorboard.
+
+Tensorboard accepts different types of inputs, so they are defined here to keep
+tensorboard separate from the rest of the logger.
+"""
+
+
+class HistogramInput:
```

Comment:
these are naked data classes, so they generally don't get unit tests. however, it's expected that they get covered through usage in other unit tests.